niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/RuntimePermissionTester.java

```diff
@@ -503,6 +535,10 @@ public void onStartFailure(int errorCode) {
                         throw new BypassTestException(
                                 "The bluetooth adapter must be enabled for this test");
                     }
+                    if (ActivityCompat.checkSelfPermission(mContext, Manifest.permission.BLUETOOTH_SCAN) != PackageManager.PERMISSION_GRANTED) {
```

Comment:
This permission check should be removed otherwise this test will never verify the path when the permission under test is not granted.

Reply:
removed. let me check it later...

Reply:
checked

Reply:
Thanks!